meson.build

```diff
@@ -0,0 +1,82 @@
+project('SU2', 'c', 'cpp', 
+        version: '6.2.0', 
+	license: 'LGPL2',
+	default_options: ['buildtype=release', 
```

Comment:
What is the default optimization level, and should the warning level not be set consistently with the Wall Wextra recommendation?

Reply:
> What is the default optimization level, and should the warning level not be set consistently with the Wall Wextra recommendation?

I added a section on advanced configuration at the link posted above. Hopefully that answers the questions.